### ISSUE TITLE
MICROSCOPY - Set XY stage movement based on FOV

### ISSUE THREAD

**Issue by iwaniec** [EtalumaSupport/LumaViewPro#65]

**Comment by kmcleaster** [comment]
One approach might be to create a "sets" of AMAX and VMAX values for the FOV of each objective so that the FOV is traversed consistently. For example if 10 seconds is a user-friendly duration to traverse the FOV, Low magnification objectives could be set to use faster speeds and accelerations and higher magnifications would use lower speed/accelerations. Similarly, in issue #64, the continuous jog positioning arrows could be set with a low acceleration, but the same VMAX to move slowly at first to find an artifact within a platewell, but by holding the control down reach the other side of the plate within a user-friendly duration.

**Comment by kmcleaster** [comment]
I am noticing that the manual Z-focus controls have their resolution modified by the objective magnification. Good! 
The X-Y stage movement is an extension of this concept.

**Comment by kmcleaster** [comment]
I am trying to determine a good set of AMAX parameters for x-y travel within the FOV of the particular objective magnification. In doing so I discovered an issue with the x-y homing routine similar to the time out issue with the Z homing routine. At very slow AMAX =170, the homing routine times out, and sets the position at zero at that point. Clicking on xy homing again successively times out until the stage limit is reached. This might not be an issue in use, since a different setting for AMAX would be set and reset depending on the operation.

**Comment by kmcleaster** [comment]
I measured the FOV of a Motic 4X objective by aligning a dust spec on a microscope slide to one edge (x axis) of the fully displayed image and moving it until the dust spec was aligned on the other edge of the display. This FOV measured 3.54mm.
I repeated this with a 20X objective and measured the FOV at 0.8mm. This is consistent with the 0.78mm 20X FOV of our earlier products. The resolution of the XY position display isn't sufficient for a more accurate assessment without a microscope measurement slide. Since FOV's for the 40X and 60X will be even smaller, there is no reason now to measure them. Setting AMAX = 170 yields an FOV edge-to-edge transit time of about 4 sec. It seems unnecessary to make changes to VMAX since the slow acceleration and short distances will never allow the velocity get near MAX. There may be no advantage to having a different AMAX for each objective.
